app/assets/json/translations/react/en.json

```diff
@@ -1578,6 +1580,12 @@
   "signIn.signInRequired": "Please sign in to your account to access this page.",
   "signIn.signedOutSuccessfully": "You have successfully logged out of your account.",
   "signIn.userTokenValidationTimeout": "We care about your security. We logged you out due to a connection issue. Please sign in to continue.",
+  "signIn.newAccount.title" : "Check your email to finish creating your account",
+  "signIn.newAccount.p1": "We sent a link to %{email}.",
+  "signIn.newAccount.p2": "Follow the link in your email to finish creating your account. It expires after 24 hours.",
+  "signIn.newAccount.sendEmailAgainButton": "Send email again",
+  "signIn.newAccount.sendEmailAgainButton.confirmation": "Email sent. Check your email.",
+  "signIn.newAccount.sendEmailAgainButton.error": "Something went wrong. We could not send an email. Try sending it again or refreshing the page.",
```

Comment:
These strings are duplicated?

Reply:
hmm let me try merging latest main

Reply:
weird, I will delete them, and then run the phrase commands again